### PR TITLE
Dat257-10 dashboard components

### PR DESCRIPTION
**Pull request by Huffla**

### CODE REVIEW

File: Client/Components/ComparisonComponent.razor.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.AspNetCore.Components;
+
+using API;
+using System.Linq;
+
+namespace Client.Components
+{
+    public partial class ComparisonComponent
+    {
+        //TODO: Make them required
```
Comment (filipkauffeldt) [inline Client/Components/ComparisonComponent.razor.cs]: Remove comment

File: Client/Components/ComparisonComponent.razor.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                }
+            }
+            return null;
+        }
+        
+        private string SetComparisonValues()
+        {
+
+           
+
```
Comment (filipkauffeldt) [inline Client/Components/ComparisonComponent.razor.cs]: Remove new lines

File: Client/Components/ComparisonComponent.razor.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+﻿
+.main{
+
```
Comment (filipkauffeldt) [inline Client/Components/ComparisonComponent.razor.css]: Remove new line

**Review by filipkauffeldt — DISMISSED** [review]

**Review by jakobschy — DISMISSED** [review]
Good

File: Client/.filenesting.json
Comment (Flaskhals68) [inline Client/.filenesting.json:1]: Should be removed with gitignore

**Review by filipkauffeldt — APPROVED** [review]

**Review by jakobschy — APPROVED** [review]

**Pull request merged**